apt.txt

```diff
@@ -1,4 +1,7 @@
+git
```

Comment:
This is nitpicking but git-core is included a few lines down. Not a problem for me though